test/parallel/test-permission-warning-flags.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+
+const warnFlags = [
+  '--allow-child-process',
+  '--allow-worker',
+];
+
+for (const flag of warnFlags) {
+  const { status, stderr } = spawnSync(
+    process.execPath,
+    [
+      '--experimental-permission', flag, '-e',
+      'setTimeout(() => {}, 1)',
+    ]
+  );
+
+  const flagWarn = stderr.toString().split('\n')[2];
+  assert.match(flagWarn, new RegExp(`SecurityWarning: The flag ${flag} must be used with extreme caution`));
```

Comment:
```suggestion
  assert.match(flagWarn, new RegExp(`^SecurityWarning: The flag ${flag} must be used with extreme caution$`));
```

Reply:
This doesn't work. The full message starts with the process:pid and ends with other message:

```
'(node:28943) SecurityWarning: The flag --allow-child-process must be used with extreme caution. It could invalidate the permission model.'
```

Reply:
Why bother with line splitting then? We might as well run the regex on the full output.

```suggestion
  assert.match(stderr.toString(), new RegExp(`SecurityWarning: The flag ${flag} must be used with extreme caution`));
```

Reply:
Fixed in 7dc06682d5